x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/mapping.ts

```diff
@@ -50,6 +51,21 @@ export const createConditionEntry = <T extends ConditionEntryField>(
   return { field, value, type: 'match', operator: 'included' };
 };
 
+export const tagsToEffectScope = (tags: string[]): EffectScope => {
+  const policyReferenceTags = tags.filter((tag) => tag.startsWith('policy:'));
+
+  if (policyReferenceTags.some((tag) => tag === 'policy:all')) {
```

Comment:
Maybe I don't have a clear understanding because this here also seems to suggest that a trusted app entry can be either global OR policy specific, but not both.  Is the global list automatic applied to all polices with the user not being able to change that?